composer.json

```diff
@@ -16,7 +16,6 @@
     "doctrine/doctrine-bundle": "^1.9|^2.0",
     "symfony/event-dispatcher": "^3.4|^4.0|^5.0",
     "symfony/security-bundle": "^3.4|^4.0|^5.0",
-    "symfony/translation": "^3.4|^4.0|^5.0",
```

Comment:
Nice catch for the duplicate ;)

Reply:
was already fixed in `master` since #155